src/material-examples/dialog-overview/dialog-overview-example-dialog.html

```diff
@@ -6,6 +6,6 @@ <h1 mat-dialog-title>Hi {{data.name}}</h1>
   </mat-form-field>
 </div>
 <div mat-dialog-actions>
-  <button mat-button [mat-dialog-close]="data.animal" tabindex="2">Ok</button>
   <button mat-button (click)="onNoClick()" tabindex="-1">No Thanks</button>
+  <button mat-button [mat-dialog-close]="data.animal" tabindex="2">Ok</button>
```

Comment:
Not something that you added, but while we're at it, can you remove the `tabindex` from the buttons here and in the other example and replace the positive `tabindex` with an `cdk-focus-initial` attribute?

Reply:
Will do @crisbeto, is there an attribute order? `<button mat-button cdk-focus-initial [mat-dialog-close]="true">` or `<button mat-button [mat-dialog-close]="true" cdk-focus-initial>`

Reply:
We don't have any rules for the attribute order.

Reply:
Ok!